panel.py

```diff
@@ -39,9 +39,13 @@ def create_panel(self):
             self.tree.insert('','0','overall mood',text='Overall Mood') 
 
             for i in range(len(self.activities)-1,-1,-1):
-                self.tree.insert('','0','activity'+str(i),text=self.activities[i]) # This inserts the activites starting at the end of the list going to beggining because an activity is displayed,
+                self.tree.insert('','0','activity'+str(i),text=self.activities[i][0]) # This inserts the activites starting at the end of the list going to beggining because an activity is displayed,
                 # Then when the next is inserted the one before is moved down 
 
+            Checkbutton(self.w, text="male", variable=self.activities[0][1]).pack(side=RIGHT)
```

Comment:
Ok so this strikes me at first glance:

1. You are creating an instance of Checkbutton, but you're not assigning it to a variable.
2. Shouldn't you be creating 7 * (number of activities) instances of the Checkbutton object ? 
2. What variables point to the empty cells in the table ?

Reply:
i was making one just so i could find out how to place and how they work

Reply:
Ok so from what I understand about TreeView, they automatically create nodes from empty space... these are the variables that should have the instance of Checkbutton object right ?

Reply:
yeh

Reply:
Ok so a for loop to those empty nodes should do right ?

Reply:
yes